.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+	"semi": true,
+	"tabWidth": 2,
```

Comment:
This one conflicts with editor conf. We use 4.

Reply:
Copied this from hrms. I think it was needed for correctly formatting CSS files that were previously indented with two spaces. But let me check again.

Reply:
ERPNext doesn't have much css, we can ignore ig.